docker-compose.yml

```diff
@@ -40,15 +40,15 @@ x-environment: &default-environment
   # Local development route used in Lagoon images and Pygmy to route requests.
   LAGOON_ROUTE: *default-url
   # Local database host (not used in production).
-  MARIADB_HOST: mariadb
+  DATABASE_HOST: database
```

Comment:
These may be non-standard variables in Lagoon, meaning this can break in Lagoon.